src/data/faq_de.json

```diff
@@ -406,8 +406,8 @@
                             "Eine Deinstallation der App sollte immer die letzte mögliche Lösung für bekannte oder neu auftretende Probleme sein.",
                             "Die App selbst kann die Covid-19-Begegnungsaufzeichnungen nicht löschen. Daher obliegt es dem Exposure Notification Framework (ENF) zu entscheiden, ob diese Daten gelöscht werden oder nicht.",
                             "In der derzeitigen Version bleiben bei einer Deinstallation unter iOS die Begegnungsaufzeichnungen erhalten. Es gibt jedoch keine Garantien, wie lange diese Daten vorgehalten werden.",
-                            "Auf Android-Geräten werden die Begegnungsaufzeichnungen gelöscht, falls die Corona-Warn-App die letzte App war, die das ENF nutzt. Falls sie zusätzlich z.B. SwissCovid oder Immuni installiert haben, bleiben die Begegnungsaufzeichnungen erhalten, bis auch diese Apps gelöscht werden oder Sie händisch eine Löschung der Begegnungsaufzeichnungen veranlassen.",
-                            "Da sich dieses Verhalten bei der App-Deinstallation auch ohne Änderungen an der Corona-Warn-App jederzeit ändern kann, empfehlen wir, wirklich nur in Ausnahmefällen die App zu löschen.",
+                            "Auf Android-Geräten werden derzeit beim Deinstallieren <a href='https://developers.google.com/android/exposure-notifications/exposure-notifications-api#methods' target='_blank' rel='noopener noreferrer'>'Datenbank und Schlüssel' vom Gerät gelöscht</a>. Sie sollten daher davon ausgehen, dass mit Löschung der App auch die Begegnungsaufzeichnungen verloren gehen.",
```

Comment:
Accordingly:
Auf Android-Geräten werden derzeit beim Deinstallieren <a href='https://developers.google.com/android/exposure-notifications/exposure-notifications-api#methods' target='_blank' rel='noopener noreferrer'>'Datenbank und Schlüssel' `durch das Exposure Notification Framework` vom Gerät gelöscht</a>.

Reply:
Same here: in my review the second line (410) seems to be deleted? That was not my intention, should stay in its place! sorry for any confusion.

Reply:
No worries, the diff view in Github is sometimes a bit confusing. I get your point :)